src/controllers/city-controller.js

```diff
@@ -0,0 +1,56 @@
+const Cities = require('../models/cities');
+const mongoose = require('mongoose');
+const {StatusCodes} = require ('http-status-codes');
+const {responseServer, responseNotFound, responseGeneral} = require('../helpers/response-result');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const {responseServer, responseNotFound, responseGeneral} = require("../helpers/response-result");
```